sdk/python/kfp/_client.py

```diff
@@ -339,6 +340,14 @@ def run_pipeline(self, experiment_id, job_name, pipeline_package_path=None, para
                                                              name=namespace,
                                                              relationship=kfp_server_api.models.ApiRelationship.OWNER)
       resource_references.append(reference)
+
+    if version_id is not None:
+      key = kfp_server_api.models.ApiResourceKey(id=version_id,
+                                                 type=kfp_server_api.models.ApiResourceType.PIPELINE_VERSION)
+      reference = kfp_server_api.models.ApiResourceReference(key=key, 
+                                                             relationship=kfp_server_api.models.ApiRelationship.OWNER)
```

Comment:
The relationship with pipeline version is creator instead of owner.

Reply:
Updated.